website/content/docs/install-boundary/initialize.mdx

```diff
@@ -63,9 +57,9 @@ To authenticate to Boundary using the recovery KMS workflow:
 
 To use the recovery workflow on the CLI, you must pass the `-recovery-config <path_to_kms_recovery_config>` flag or set the environment
 variable for `BOUNDARY_RECOVERY_CONFIG` for every command ran. Authentication takes place for every command
-ran when using the recovery workflow, there is no `boundary authenticate` step:
+ran when using the recovery workflow. `boundary authenticate` does not apply when using the recovery KMS.
```

Comment:
```suggestion
ran when using the recovery workflow. The `boundary authenticate` command does not apply when you use the recovery KMS.
```

Reply:
I usually try not to start a sentence with a command, I think it can be a little difficult to read.